python/visualization.py

```diff
@@ -615,6 +629,34 @@ def plot_eps(
                 colorbar_parameters=colorbar_parameters,
             )
 
+        if label_geometry:
+            for el in sim.geometry:
+                if sim_size.x == 0:
+                    center_first = el.center.y
+                    center_second = el.center.z
+                    sim_first = sim_center.y
+                    sim_second = sim_center.z
+                elif sim_size.y == 0:
+                    center_first = el.center.y
+                    center_second = el.center.z
+                    sim_first = sim_center.y
+                    sim_second = sim_center.z
```

Comment:
This is a bug. The set of lines within `elif sim_size.y == 0` should be:
```
                    center_first = el.center.x
                    center_second = el.center.z
                    sim_first = sim_center.x
                    sim_second = sim_center.z
```
However, note that the first two `if` / `elif` statements in this series only apply when taking a 2D cross section of a 3D cell volume corresponding to `sim.dimensions` of 3.

For the case of `sim.dimensions == 2`, `sim_size.x` and `sim_size.y` must be nonzero.

Reply:
Sorry it take a while for me to realize that I missed this comment :( 
I fixed this mistake.
I am not sure what I should do about validation of the size in the case of 2D system